src/aoc_cj/aoc2015/day19.py

```diff
@@ -1,96 +1,85 @@
-from collections import defaultdict
+import itertools
+import re
+from dataclasses import dataclass
 
-from lark import Lark
+import lark
+from more_itertools import bucket
```

Comment:
not sure how the changes to this file got in here... it certainly doesn't belong in this PR.